sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/StringValue.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import java.util.List;
+
+/**
+ * Class to represent the Float value for
+ * {@link com.azure.ai.formrecognizer.implementation.models.FieldValue#getValueString()}
+ */
+public class StringValue extends FieldValue<String> {
+    /*
+     * String value.
+     */
+    private String valueString;
+
+    /**
+     * Constructs a StringValue.
+     *
+     * @param text The text content of the extracted field.
+     * @param boundingBox Bounding box of the field value.
+     * @param valueString String value.
+     */
+    public StringValue(String text, BoundingBox boundingBox, String valueString) {
+        super(text, boundingBox);
+        this.valueString = valueString;
+    }
+
+    @Override
+    public String getValue() {
+        return this.valueString;
+    }
+
+    @Override
+    public void setValue(String value) {
+        this.valueString = value;
+    }
+
+    @Override
+    public List<Element> getElements() {
+        return super.getElements();
+    }
+
+    @Override
+    public Float getConfidence() {
+        return super.getConfidence();
```

Comment:
Javadoc missing for these methods. Also, look at other model classes in public package to ensure all of them have javadocs.